foursight_core/deploy.py

```diff
@@ -90,7 +122,7 @@ def build_config(cls, stage, identity=None, stack_name=None,
             # env_name = None
             # s3_key_id = None  # not supported in legacy
             # if not (s3_enc_secret and client_id and client_secret and dev_secret):
-            #     print(''.join(['ERROR. You are missing one more more environment ',
+            #     PRINT(''.join(['ERROR. You are missing one more more environment ',
             #                    'variables needed to deploy Foursight.\n',
             #                    'Need: S3_ENCRYPT_KEY, CLIENT_ID, CLIENT_SECRET, DEV_SECRET.'])
             #           )
```

Comment:
```suggestion
            #     PRINT('ERROR. You are missing one more more environment'
            #           ' variables needed to deploy Foursight.'
            #           ' S3_ENCRYPT_KEY, CLIENT_ID, CLIENT_SECRET, DEV_SECRET are required.')
```

Reply:
Thanks, but this is commented out code; should remove it.